cookbooks/continental_extension/doc/continental_extension.md

```diff
@@ -1,39 +1,39 @@
 # Continental extension
 
 *This section was contributed by John Naliboff, Anne Glerum, and Valentina
-Magni*
+Magni *
 
 In the crustal deformation examples above, the viscosity depends solely on the
 Drucker Prager yield criterion defined by the cohesion and internal friction
 angle. While this approximation works reasonably well for the uppermost crust,
-deeper portions of the lithosphere may undergo either brittle or viscous
-deformation, with the latter depending on a combination of composition,
-temperature, pressure and strain-rate. In effect, a combination of the
-Drucker-Prager and Diffusion dislocation material models is required. The
-visco-plastic material model is designed to take into account both brittle
-(plastic) and non-linear viscous deformation, thus providing a template for
-modeling complex lithospheric processes. Such a material model can be used in
-using the following set of input parameters:
-
-``` prmfile
-```
-
-This cookbook provides one such example where the continental lithosphere
-undergoes extension. Notably, the model design follows that of numerous
-previously published continental extension studies&nbsp;(Huismans and Beaumont
-2011; Brune et al. 2014; Naliboff and Buiter 2015, and references therein).
+       deeper portions of the lithosphere may undergo either brittle or viscous
+       deformation, with the latter depending on a combination of composition,
+       temperature, pressure and strain-rate. In effect, a combination of the
+       Drucker-Prager and Diffusion dislocation material models is required. The
+       visco-plastic material model is designed to take into account both brittle
+       (plastic) and non-linear viscous deformation, thus providing a template for
+         modeling complex lithospheric processes. Such a material model can be used in
+       using the following set of input parameters:
+
+         ``` {literalinclude} continental_extension_material_model.prm
```

Comment:
There's something weird going on with the indentation that's causing a lot of these directives to not build properly. It looks like you intentionally added these indents in the latest commit, can I ask why?

Reply:
@cmills1095 - sorry about that. `make indent` was altering additional files aside from this one, so I tried running style on just this file. That clearly did not work, and the update commit hopefully has the correct indentation.